### PR TITLE
Added valencian variant for catalan language

### PR DESCRIPTION
**Pull request by joamuran**

### CODE REVIEW

**Comment by jmoenig** [comment]
wow, cool! Thank you.

File: lang-ca_valencia.js [hunk 1]
```diff
@@ -0,0 +1,1394 @@
...
+    'parent':
+        'pare',
+    'children':
+        'fill',
+    'clones':
+        'clons',
+    'other clones':
+        'altres clons',
+    'dangling?':
+        'Baumeln?',
```
Comment (bromagosa) [inline lang-ca_valencia.js]: Açò no me pareix pas valencià ;)

Comment (jmoenig) [reply lang-ca_valencia.js]: Well, that's the Stuttgart variant of the Valencian variant of Catalan :-D

Comment (bromagosa) [reply lang-ca_valencia.js]: There are so many German tourists that you guys are starting to influence the language!

@joamuran com ho traduiries tu? En @jguille2 ha posat "penjat?" en valencià del nord :)

**Pull request merged**